examples/gallery/demos/matplotlib/bachelors_degrees_by_gender.ipynb

```diff
@@ -106,14 +107,12 @@
     "    ax.spines['bottom'].set_visible(False)\n",
     "    ax.spines['left'].set_visible(False)\n",
     "\n",
-    "# Define various options to adjust the plot\n",
-    "options = hv.Store.options(backend='matplotlib')\n",
-    "options.Curve = hv.Options('plot', show_frame=False, bgcolor='white', labelled=[], show_grid=False,\n",
-    "                           aspect=0.7, show_legend=False, xticks=5, final_hooks=[cb], fig_size=350)\n",
-    "options.Curve = hv.Options('style', color=hv.Cycle(values=color_sequence), linewidth=2)\n",
-    "\n",
-    "(curves.redim(Year=dict(range=(1970, 2011))) *\n",
-    " labels.options(color_index='Degree', cmap=color_sequence)).relabel(title)"
+    "(curves * labels).relabel(title).opts(\n",
```

Comment:
Again, why not use the `title`/`title_format` option? I don't like complex expressions before `.opts`.